akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/StashBuffer.scala

```diff
@@ -94,6 +94,8 @@ object StashBuffer {
    * It's allowed to stash messages while unstashing. Those newly added
    * messages will not be processed by this call and have to be unstashed
    * in another call.
+   *
+   * The initial `behavior` passed to `unstashAll` must not be `unhandled`
```

Comment:
Are we afraid of `.` at the end of a doc sentence that is not an explicit `@param` description?

Reply:
Nah, just sloppy I think